lib/helper.coffee

```diff
@@ -0,0 +1,91 @@
+ChildProcess = require 'child_process'
+Path = require 'path'
+FS = require 'fs'
+{XRegExp} = require 'xregexp'
+
+class Helpers
+  @findFile: (directory, fileName) ->
```

Comment:
As a side note, this is also perfectly valid:
```coffeescript
module.exports = Utils =
  functionOne: () ->
    Utils.functionTwo()
  functionTwo: () ->
    2
```
Personally, I try to avoid stateless classes, just to make intent clear (if it's a class, it has state)